README.md

```diff
@@ -57,8 +57,8 @@ This is quite slow and can take up to one or two hours but can provide good resu
 -   mcc
 -   functions
 -   classes
--   lines_of_code - The number of lines inside a file
--   comment_lines (not unit tested)
+-   lines_of_code - The number of lines of a file
+-   comment_lines - The number of comment lines inside a file
```

Comment:
Let's define the metrics more precisely.
It should be clear as much as possible what a metric is counting:
- Lines of Code = The number of lines of a file including empty lines.
- And comment_lines = the number of comment lines inside a file. Doc Blocks, empty comment lines, comments in same line as code, etc.

I think we should optimize this for all metrics (maybe an a follow up PR) by providing proper metric definitions.